### PR TITLE
enh(release): Add Centreon Web 20.10.13

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [ ] 20.04.x
- [x] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x (master)

### CODE REVIEW

**Review by mdebauchez — APPROVED** [review]

File: en/releases/centreon-core.md [hunk 1]
```diff
@@ -19,6 +19,39 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 20.10.13
+
```
Comment (tuntoja) [inline en/releases/centreon-core.md:23]:
```suggestion

Release date: 'November 10, 2021'

```

File: fr/releases/centreon-core.md [hunk 2]
```diff
@@ -19,6 +19,39 @@ commerciales, vous pouvez vous rendre sur notre
 
 ## Centreon Web
 
+### 20.10.13
+
```
Comment (tuntoja) [inline fr/releases/centreon-core.md:23]:
```suggestion

Release date: '10 novembre 2021'

```

**Review by tuntoja — CHANGES_REQUESTED** [review]

File: en/releases/centreon-core.md (hunk 1 above)
Comment (tuntoja) [reply en/releases/centreon-core.md:23]: added release date

File: fr/releases/centreon-core.md (hunk 2 above)
Comment (tuntoja) [reply fr/releases/centreon-core.md:23]: added release date

**Review by tuntoja — CHANGES_REQUESTED** [review]

**Review by tuntoja — APPROVED** [review]

File: en/releases/centreon-core.md [hunk 3]
```diff
@@ -19,6 +19,41 @@ If you have feature requests or want to report a bug, please go to our
...
+Release date: 'November 10, 2021'
```
Comment (cg-tw) [inline en/releases/centreon-core.md:24]:
```suggestion
Release date: `November 10, 2021`
```

File: fr/releases/centreon-core.md [hunk 4]
```diff
@@ -19,6 +19,41 @@ commerciales, vous pouvez vous rendre sur notre
...
+Release date: '10 novembre 2021'
```
Comment (cg-tw) [inline fr/releases/centreon-core.md:24]:
```suggestion
Release date: `10 novembre 2021`
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: en/releases/centreon-core.md [hunk 5]
```diff
@@ -19,6 +19,41 @@ If you have feature requests or want to report a bug, please go to our
...
+#### Enhancements
+
+- [CEIP] Product Adoption component integration
+- [Core] Add Feature Flipping for Resources Status vs Legacy Pages
+- [Core] Compatibility PHP 7.4
+
+#### Bug fixes
+
+- [Authentication] Fixed user information retrieval method for OpenId
+- [CLAPI] Fixed recurrent downtimes exported with CLAPI cannot be imported with CLAPI
```
Comment (cg-tw) [inline en/releases/centreon-core.md:35]:
```suggestion
- [CLAPI] Fixed recurrent downtimes exported with CLAPI that could not be imported with CLAPI
```

File: en/releases/centreon-core.md [hunk 6]
```diff
@@ -19,6 +19,41 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Configuration] Avoid blocking configuration generation when user don't have notifications enabled
```
Comment (cg-tw) [inline en/releases/centreon-core.md:36]:
```suggestion
- [Configuration] Avoid blocking configuration generation when users don't have notifications enabled
```

File: en/releases/centreon-core.md [hunk 7]
```diff
@@ -19,6 +19,41 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Configuration] Massive changes on hosts reset untouched values
+- [Configuration] Unwanted empty string parameter for LUA output in Broker configuration
```
Comment (cg-tw) [inline en/releases/centreon-core.md:38]:
```suggestion
- [Configuration] Fixed unwanted empty string parameter for LUA output in Broker configuration
```

File: en/releases/centreon-core.md [hunk 8]
```diff
@@ -19,6 +19,41 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Custom Views] Can't share views with LDAP groups
```
Comment (cg-tw) [inline en/releases/centreon-core.md:39]:
```suggestion
- [Custom Views] Couldn't share views with LDAP groups
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by tuntoja — APPROVED** [review]

File: en/releases/centreon-core.md [hunk 9]
```diff
@@ -19,6 +19,41 @@ If you have feature requests or want to report a bug, please go to our
...
+
+#### Bug fixes
+
+- [Authentication] Fixed user information retrieval method for OpenId
+- [CLAPI] Fixed recurrent downtimes exported with CLAPI that could not be imported with CLAPI
+- [Configuration] Avoid blocking configuration generation when users don't have notifications enabled
+- [Configuration] Fixed reset untouched values using massive changes on hosts
+- [Configuration] Fixed empty string parameter for LUA output in Broker configuration
+- [Custom Views] Fixed sharing of views with LDAP groups
+- [Discovery] Error message when filling credentials in job as side-effect of a change regarding downtimes
```
Comment (tuntoja) [inline en/releases/centreon-core.md:40]:
```suggestion
- [Discovery] Fixed an issue causing error messages when filling credentials in a Host Discovery job
```

File: fr/releases/centreon-core.md [hunk 10]
```diff
@@ -19,6 +19,41 @@ commerciales, vous pouvez vous rendre sur notre
...
+
+#### Bug fixes
+
+- [Authentication] Fixed user information retrieval method for OpenId
+- [CLAPI] Fixed recurrent downtimes exported with CLAPI that could not be imported with CLAPI
+- [Configuration] Avoid blocking configuration generation when users don't have notifications enabled
+- [Configuration] Fixed reset untouched values using massive changes on hosts
+- [Configuration] Fixed empty string parameter for LUA output in Broker configuration
+- [Custom Views] Fixed sharing of views with LDAP groups
+- [Discovery] Error message when filling credentials in job as side-effect of a change regarding downtimes
```
Comment (tuntoja) [inline fr/releases/centreon-core.md:40]:
```suggestion
- [Discovery] Fixed an issue causing error messages when filling credentials in a Host Discovery job
```

**Review by tuntoja — CHANGES_REQUESTED** [review]

File: en/releases/centreon-core.md [hunk 11]
```diff
@@ -19,6 +19,41 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 20.10.13
+
+Release date: `November 10, 2021`
```
Comment (tuntoja) [inline en/releases/centreon-core.md:24]:
```suggestion
```

**Review by tuntoja — APPROVED** [review]

**Review by tuntoja — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**